cluster/crds/pkg.crossplane.io_locks.yaml

```diff
@@ -61,6 +61,12 @@ spec:
                         description: Package is the OCI image name without a tag or
                           digest.
                         type: string
+                      parentConstraints:
```

Comment:
Is this expected? So, will we start seeing this field in the lock object on the kube and parentConstraints persisted?